### ISSUE TITLE
[BUG] New or deleted accounts receive "unable to sign in" error

### ISSUE THREAD

**Issue by nelshopp** [rokwire/illinois-app#3304]
**Describe the bug**
Customers with new or deleted accounts receive "unable to sign in" error.  

**To Reproduce**
1. 'Delete my account' under settings (or have a new university student or staff account)
2. Run Illinois app
3. Attempt to log in

We currently have eight customers who are receiving "unable to log in. Please try again later".  

This has been confirmed by Kim.  Most are running iPhones but Kim confirmed it affects Android as well (it worked for my Android).  

 
**Screenshots**
![image](https://github.com/rokwire/illinois-app/assets/91676531/24c42df8-b9e0-4b5f-91f3-03743d734d54)

**Comment by knguyenjahiel** [comment]
Below is a recording of the sequence that gave me the "Unable to sign in" error message. I deleted my account (deleted All My Information"). This occurs on both iPhone and Android. Note the brief Shibboleth error message, at 0:00:06. Similar "Unable to sign in" message occurs when I did a clean install of the app.

https://github.com/rokwire/illinois-app/assets/118769169/f41df814-9d7f-4ee5-a8fa-033cb0b13d26
